coriolis/conductor/rpc/server.py

```diff
@@ -331,7 +331,7 @@ def _check_create_reservation_for_transfer(
             transfer_type, ninstances)
         transfer_action.reservation_id = reservation['id']
 
-    def _check_reservation_for_transfer(self, transfer_action):
+    def _check_reservation_for_transfer(self, transfer_action, transfer_type):
```

Comment:
Considering the `transfer_type` param specifically expects one of the `licensing.RESERVATION_TYPE_*` constants, please rename this arg to `reservation_type`.